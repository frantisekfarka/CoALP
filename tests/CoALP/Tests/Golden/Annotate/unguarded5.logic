connected(X,Y):-connected(Z,Y),edge(X,Z).
connected(X,X).
