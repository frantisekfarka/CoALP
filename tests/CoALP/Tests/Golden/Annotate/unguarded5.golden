connected(V_2i, V_3i, transform-func-1(V_6c, V_7i)) :- connected(V_4i, V_3i, V_6c), edge(V_2i, V_4i, V_7i).
connected(V_5i, V_5i, transform-func-2).
